Products/ZenModel/actions.py

```diff
@@ -394,6 +396,37 @@ def _encodeBody(self, body):
             plain_body = MIMEText(body.decode('ascii', 'ignore'))
         return plain_body
 
+    def _evtTimeToUserTz(self, body, zone_name):
+        """
+        Convert event time from email body to timezone from use property.
+        """
+        evt_date = ''
+        evt_date_with_tz = ''
+        try:
+            evt_date = re.search(
+                '(\d+/\d+/\d+ \d+:\d+:\d+.\d+)', body).group()
```

Comment:
I saw them in every events that came for notification. I can take them from there, since in the result I'm not setting them anyway.